Setup/UpgradeData.php

```diff
@@ -1,120 +0,0 @@
-<?php
-
-namespace Razorpay\Subscription\Setup;
-
-use Magento\Catalog\Model\Product;
-use Magento\Eav\Setup\EavSetupFactory;
-use Magento\Framework\Setup\ModuleContextInterface;
-use Magento\Framework\Setup\ModuleDataSetupInterface;
-use Magento\Framework\Setup\UpgradeDataInterface;
-use Razorpay\Subscription\Model\SubscriptionProductAttributes;
-
-class UpgradeData implements UpgradeDataInterface
```

Comment:
Why this file is deleted. whats the alternative to subscription config in product creation page

Reply:
as it no long support